gcloud-java-dns/src/main/java/com/google/gcloud/dns/DnsBatch.java

```diff
@@ -150,7 +152,7 @@ public void submit() {
     return ImmutableMap.copyOf(temp);
   }
 
-  private DnsRpc.Callback<ManagedZonesListResponse> listZonesCallback(
+  private DnsRpc.Callback<ManagedZonesListResponse> newListZonesCallback(
```

Comment:
I would vote for `create` then

I think that makes it more clear when a method does not do anything other than creating and return instance of X.